### ISSUE TITLE
Cuda error for kernel image

### ISSUE THREAD

**Issue by bamdad-b** [cassiebreviu/StableDiffusion#25]
Hi, thanks for the great project.
I can run the project in CPU mode, but when I switch to cuda, the following error is thrown on running text encoding session:

> Microsoft.ML.OnnxRuntime.OnnxRuntimeException: '[ErrorCode:Fail] Non-zero status code returned while running Slice node. Name:'Slice_218' Status Message: CUDA error cudaErrorNoKernelImageForDevice:no kernel image is available for execution on the device'

I'm using Windows 10, nvidia GT 710

**Comment by LeonNerd** [comment]
hi ,i have an error when run the project in CPU mode.
My tool is vs2022，net6.0, I can rebuild the project from sln
When I configured the model path and chose to run cpu only，an error will occur.
'''
Microsoft.ML.OnnxRuntime.OnnxRuntimeException:“[ErrorCode:InvalidArgument] Tensor element data type discovered: Int64 metadata expected: Float”
'''

'''
I checked the code location is UNet.cs line105
input = CreateUnetModelInput(textEmbeddings, latentModelInput, timesteps[t]);
// Run Inference
var output = unetSession.Run(input);
'''
input is a list with three objects.I don't know why this mistake happened,I guess it's because of the onnxruntime version.
But the onnxruntime is built automatically by sln
